sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/TableAsyncClientTest.java

```diff
@@ -1052,14 +1052,13 @@ public void generateSasTokenWithAllParameters() {
     }
 
     @Test
-    // Disabling as this currently fails and prevents merging https://github.com/Azure/azure-sdk-for-java/pull/28522.
-    // TODO: Will fix in a separate PR. -vicolina
     public void canUseSasTokenToCreateValidTableClient() {
         // SAS tokens at the table level have not been working with Cosmos endpoints.
         // TODO: Will re-enable once the above is fixed. -vicolina
         Assumptions.assumeFalse(IS_COSMOS_TEST, "Skipping Cosmos test.");
 
-        final OffsetDateTime expiryTime = OffsetDateTime.of(2023, 12, 12, 0, 0, 0, 0, ZoneOffset.UTC);
+        // This offset time cannot be a past date. Will cause the test to fail if it is.
+        final OffsetDateTime expiryTime = OffsetDateTime.of(2024, 12, 12, 0, 0, 0, 0, ZoneOffset.UTC);
```

Comment:
We should update this to take today's date (`OffsetDateTime.now()`) and then add some arbitrary amount of time to it instead of hard-coding some datetime.